keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -254,24 +253,43 @@ func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]stri
 		Image: &runtimeapi.ImageSpec{
 			Image: edgeImage,
 		},
+		// Keep the container running by passing in a command that never ends.
+		// so that we can ExecSync in the following operations,
+		// to ensure that we can copy files from container to host totally and correctly
 		Command: []string{
 			"/bin/sh",
 			"-c",
-			copyCmd,
+			"sleep infinity",
 		},
 		Mounts: mounts,
 	}
 	containerID, err := runtime.RuntimeService.CreateContainer(sandbox, containerConfig, psc)
 	if err != nil {
-		return err
+		return fmt.Errorf("create container failed: %v", err)
 	}
 	defer func() {
 		if err := runtime.RuntimeService.RemoveContainer(containerID); err != nil {
```

Comment:
When the copycmd run successfully, the container will be killed here, right?

Reply:
yes